content/posts/2023/reana-0.9.1.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "REANA 0.9.1 is released"
+date: 2023-09-28T09:00:00+01:00
+---
+
+We are happy to announce REANA 0.9.1, a minor update that allows users to preview PDF and ROOT files from the web interface, makes it easier to clean up old workflows to free disk space, adds support for integrating with Keycloak, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### PDF/ROOT preview
+
+REANA 0.9.1 improves the preview of workspace files from the web interface by adding support for PDF and ROOT files.
+You will be able to interact with ROOT plots directly from your browser, without the need to download the files on your computer.
+
+![ROOT file preview](/images/reana-0.9.1-root-preview.png)
+
+### Pruning workspaces
+
+REANA already provides tools to keep disk usage under control, for example by setting up automatic retention rules or by deleting old workflows.
```

Comment:
... to keep the disk usage under control

... for example by setting up automatic file retention rules

The latter could be linked to the blog post about it https://blog.reana.io/posts/2022/workspace-file-retention-rules/